readme.markdown

```diff
@@ -158,6 +159,10 @@ Number of milliseconds during which clicks where the mouse button is released wi
 
 You can provide an `invalid` method with a `(el, target)` signature. This method should return `true` for elements that shouldn't trigger a drag. Here's the default implementation, which prevents drags originating from anchor elements and buttons.
 
+#### `options.mirrorContainer`
+
+You can provide an `Node` that will be the parent of all generated mirror nodes.
```

Comment:
Just state DOM element instead of `Node`. Let's use this copy:

> The DOM element where the mirror element displayed while dragging will be appended to. Defaults to `document.body`.